src/main/java/com/exadel/discountwebapp/category/controller/CategoryController.java

```diff
@@ -0,0 +1,45 @@
+package com.exadel.discountwebapp.category.controller;
+
+import com.exadel.discountwebapp.category.entity.Category;
+import com.exadel.discountwebapp.category.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/category")
+public class CategoryController {
+
+    private final CategoryService categoryService;
+
+    @Autowired
+    public CategoryController(CategoryService categoryService) {
+        this.categoryService = categoryService;
+    }
+
+    @GetMapping({"", "/"})
```

Comment:
({"", "/"}) can be removed
just use @GetMapping

Reply:
done